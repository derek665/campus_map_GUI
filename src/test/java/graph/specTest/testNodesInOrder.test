# test if all the nodes are printed in alphabetical order

# Create graph
CreateGraph alp

# Add Nodes to graph
AddNode alp a
AddNode alp c
AddNode alp d
AddNode alp b

# Print the nodes
ListNodes alp