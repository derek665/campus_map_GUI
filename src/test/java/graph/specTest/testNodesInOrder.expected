# test if all the nodes are printed in alphabetical order

# Create graph
created graph alp

# Add Nodes to graph
added node a to alp
added node c to alp
added node d to alp
added node b to alp

# Print the nodes
alp contains: a b c d